rj_constants/include/rj_constants/topic_names.hpp

```diff
@@ -32,3 +34,56 @@ constexpr auto kDetectionFramePub = "vision_receiver/detection_frame";
 namespace vision_filter::topics {
 constexpr auto kWorldStatePub = "vision_filter/world_state";
 }  // namespace vision_filter::topics
+
+namespace gameplay::topics {
+
+static inline std::string robot_intent_pub(int robot_id) {
+    return "robot/" + std::to_string(robot_id) + "/gameplay/robot_intent";
+}
+
+}  // namespace gameplay::topics
+
+namespace planning::topics {
+
+constexpr auto kGlobalObstaclesPub = "planning/global_obstacles";
+constexpr auto kGoalZoneObstacles = "planning/goal_zone_obstacles";
+
+static inline std::string trajectory_pub(int robot_id) {
+    return "robot/" + std::to_string(robot_id) + "/planning/trajectory";
+}
+
+}  // namespace planning::topics
+
+namespace control {
+
+namespace topics {
+
+static inline std::string manipulator_setpoint_pub(int robot_id) {
+    return "robot/" + std::to_string(robot_id) + "/control/manipulator_setpoint";
+}
```

Comment:
What's manipulator? We have manipulators?

Reply:
Manipulator = everything that's not the drivetrain.

I didn't have a better name for it.

Reply:
so like the kicker / chipper?